bundles/net.tourbook.statistics/src/net/tourbook/statistics/graphs/StatisticDay.java

```diff
@@ -593,6 +604,20 @@ public void saveState(final IDialogSettings viewState) {
             viewState.put(STATE_SELECTED_TOUR_ID, Long.toString(_statisticData_Day.allTourIds[valueIndex]));
          }
       }
+
+      if (_athleteBodyWeight_YData != null) {
+
+         _prefStore.setValue(ITourbookPreferences.STAT_DAY_BODYWEIGHT_YAXIS_MIN_VISIBLE_VALUE,
+               _athleteBodyWeight_YData.getVisibleMinValue() / UI.UNIT_VALUE_WEIGHT);
+         _prefStore.setValue(ITourbookPreferences.STAT_DAY_BODYWEIGHT_YAXIS_MAX_VISIBLE_VALUE,
+               _athleteBodyWeight_YData.getVisibleMaxValue() / UI.UNIT_VALUE_WEIGHT);
+      }
+
+      if (_athleteBodyFat_YData != null) {
+
+         _prefStore.setValue(ITourbookPreferences.STAT_DAY_BODYFAT_YAXIS_MIN_VISIBLE_VALUE, _athleteBodyFat_YData.getVisibleMinValue());
```

Comment:
@wolfgang-ch 
I have created a preference for each statistics type (i.e: Day vs Month vs Week vs Year). Maybe I could only do one preference shared across all the 4 statistics since it displays the same data (body weight and fat) ? thoughts ?

Reply:
Can you attach a screenshot

Reply:
I'm not sure i understand. I am mentioning here if I should keep the preference for each statistic type:
STAT_**DAY**_BODYFAT_YAXIS_MIN_VISIBLE_VALUE
STAT_**WEEK**_BODYFAT_YAXIS_MIN_VISIBLE_VALUE
STAT_**MONTH**_BODYFAT_YAXIS_MIN_VISIBLE_VALUE
STAT_**YEAR**_BODYFAT_YAXIS_MIN_VISIBLE_VALUE

or if I should have a unique one like this:
STAT_BODYFAT_YAXIS_MIN_VISIBLE_VALUE

Reply:
When the value is about the same in each statistic, then one value is sufficient

Reply:
Fixed. Now, one preference only is used for day/week/month/year athlete's data statistics